src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReferralHelper.java

```diff
@@ -0,0 +1,352 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.helpers;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.collections4.CollectionUtils;
+import uk.gov.hmcts.ecm.common.helpers.UtilHelper;
+import uk.gov.hmcts.et.common.model.bulk.types.DynamicFixedListType;
+import uk.gov.hmcts.et.common.model.bulk.types.DynamicValueType;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.items.DateListedTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.DocumentTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.HearingTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.ReferralReplyTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.ReferralTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.ReferralReplyType;
+import uk.gov.hmcts.et.common.model.ccd.types.ReferralType;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+
+@Slf4j
+public final class ReferralHelper {
+
+    private ReferralHelper() {
+    }
+
+    private static final String TRUE = "True";
+    private static final String FALSE = "False";
+    private static final String JUDGE_ROLE_ENG = "caseworker-employment-etjudge-englandwales";
+    private static final String JUDGE_ROLE_SCOT = "caseworker-employment-etjudge-scotland";
+    private static final String GUIDANCE_DOC_LINK = "<hr>To help you complete this form, open the "
+        + "<a href=\"url\">referral guidance documents</a>";

```

Comment:
Should this be an actual link? Also, the ticket says it should open in a new tab (target=\"_blank\")

Reply:
I have chased them for the update of the actual link, will add it here once ready. I will add the target blank for next tab